test/unit/saveArticles.test.ts

```diff
@@ -73,49 +73,37 @@ describe('saveArticles', () => {
 
       await downloader.checkCapabilities()
       await downloader.setBaseUrls()
-      const _articleDetailsRet = await downloader.getArticleDetailsIds(['Western_Greenland'])
+      const _articleDetailsRet = await downloader.getArticleDetailsIds(['User:VadimKovalenkoSNF'])
       const articlesDetail = mwRetToArticleDetail(_articleDetailsRet)
       const { articleDetailXId } = redisStore
       articleDetailXId.setMany(articlesDetail)
-      ;[{ html: articleHtml }] = await downloader.getArticle('Western_Greenland', dump, articleDetailXId)
+      ;[{ html: articleHtml }] = await downloader.getArticle('User:VadimKovalenkoSNF', dump, articleDetailXId)
       dump2 = new Dump('', { keepEmptyParagraphs: true } as any, dump.mwMetaData)
     })
 
-    test('Found no empty details elements when they should be stripped in mobile view', async () => {
+    test('Found no empty paragraph elements when they should be stripped', async () => {
       const doc = domino.createDocument(articleHtml)
       await applyOtherTreatments(doc, dump)
-
-      const details = Array.from(doc.querySelectorAll('details'))
-      let fewestChildren = 0
-      for (const d of details) {
-        if (fewestChildren === 0 || d.children.length < fewestChildren) {
-          fewestChildren = d.children.length
-        }
-      }
-      expect(fewestChildren).toBeGreaterThan(0)
+      const paragraphs = Array.from(doc.querySelectorAll('p'))
+      expect(paragraphs.length).toEqual(2)
```

Comment:
Here same feedback as earlier, the feauture seems to be not understood.

Reply:
I rollbacked tests and commented on them. We'll refer to this functionality later.